package.json

```diff
@@ -1,6 +1,6 @@
 {
     "scripts": {
-        "ds": "run-p dev:*",
+        "ds": "DUMMYURL='http://localhost:8080' run-p dev:*",
```

Comment:
The DUMMYURL environment variable should have the same value independent of which script was used to start the back-end. Therefore, it should be set in backend/.env file and not here.